.github/workflows/submitty_ci.yml

```diff
@@ -110,6 +114,7 @@ jobs:
 
 
   php-lint:
+    name: PHP Lint & Static Analysis
```

Comment:
```suggestion
    name: PHP Lint
```

Reply:
If we are fixing this, I think we also should fix python.

Reply:
Yep, good catch.  I missed that one.